bitcoin/src/bip32.rs

```diff
@@ -917,6 +917,8 @@ impl std::error::Error for InvalidBase58PayloadLengthError {}
 #[cfg(test)]
 mod tests {
     use hex::test_hex_unwrap as hex;
+    #[cfg(feature = "serde")]
```

Comment:
Side note: it'd be nice if dev-dependencies didn't have to contain serde stuff unconditionally but I don't think this is possible with Cargo today.